pages/docs/sdk/serve.mdx

```diff
@@ -397,6 +397,50 @@ ff.http('inngest', serve(inngest, [fnA]));
 
 You can run this locally with `npx @google-cloud/functions-framework --target=inngest` which will serve your Inngest functions on port `8080`.
 
+## Framework: Firebase Cloud Functions
+
+Based on the Google Cloud Function architecture, the Firebase Cloud Functions provide a different API to serve functions using `onRequest`:
+
+```typescript
+import { onRequest } from "firebase-functions/v2/https";
+
+import { serve } from "inngest/express";
+import { inngest as inngestClient } from "./inngest/client";
+
+export const inngest = onRequest(
+  serve({
+    client: inngestClient,
+    functions: [/* ...functions... */],
+  })
+);
+```
+
+<Callout>
+
+Firebase Cloud Function requires to configure the `INNGEST_SERVE_PATH` with the custom function path.
+
+For example, for a project named `inngest-firebase-functions` deployed on the `us-central1` region, the `INNGEST_SERVE_PATH` value will be as follows:
+
+```
+/inngest-firebase-functions/us-central1/inngest/
+```
+
+</Callout>
+
+You can now server your Firebase Cloud Function locally using the following command:
```

Comment:
```suggestion
To serve your Firebase Cloud Function locally, use the following command:
```

"You can now" sounds conversational and given that this is not a tutorial but a reference page, we provide directions.